templates/node.conf.2.j2

```diff
@@ -0,0 +1,51 @@
+basedir : "{{ corda_dir_location }}"
+p2pAddress : "{{ corda_host_p2p }}:{{ corda_port_p2p }}"
+rpcAddress : "{{ corda_host_rpc }}:{{ corda_port_rpc }}"
+webAddress : "0.0.0.0:{{ corda_port_web }}"
+h2port : {{ corda_port_h2 }}
+emailAddress : "{{ corda_admin_email }}"
+myLegalName : "O={{ corda_org }}, L={{ corda_city }}, C={{ corda_country }}"
+keyStorePassword : "cordacadevpass"
+trustStorePassword : "trustpass"
```

Comment:
It might be worth making this (trustStorePassword) a variable in all these configs